examples/reference/panes/HoloViews.ipynb

```diff
@@ -6,41 +6,70 @@
    "metadata": {},
```

Comment:
I'm not able to test this notebook in Jupyterlab because I started getting.

![image](https://github.com/holoviz/panel/assets/42288570/2b347bd6-113b-418c-aab9-379afea19ac6)

I've tried everything to get rid of it. But cannot.

panel==1.3.4, bokeh==3.3.2, param=2.0.1, holoviews==1.18.1, hvplot=0.9.0

Reply:
Can't reproduce anymore.